docs/api/process_adjustment_reversal.md

```diff
@@ -0,0 +1,38 @@
+This endpoint is used to process a reversal of a Sales Adjustment at the point-of-sale. 
+
+**Method:** *ProcessSalesAdjustmentReversal*
+
+<h3>Request</h3>
+
+Parameter &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;| Type | Description
+-----------|------|-------------
+x_pos_transaction_ref | Unicode string | This is the transaction reference of the adjustment reversal
+x_adjustment_signature | Unicode string | The original adjustment signature that we are trying to reverse.
```

Comment:
Why do we need the signature of the original adjustment?

Reply:
This is how we reference the original adjustment as we dont have anything else to go by. If we had more time we might do things differently.